FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/DataProcessor/ParquetDataProcessor.cs

```diff
@@ -103,12 +99,22 @@ private ParquetConverter ParquetConverter
 
             string inputContent = string.Join(
                 Environment.NewLine,
-                processedData.Values.Select(jsonObject => jsonObject.ToString(Formatting.None))
-                         .Where(result => CheckBlockSize(processParameters.SchemaType, result)));
+                processedData.Values.Select(jsonObject => {
+                    var str = jsonObject.ToString(Formatting.None);
+
+                    // add a log for large resource
+                    if (str.Length > LargeResourceSize)
+                    {
+                        _diagnosticLogger.LogInformation($"The resource size {str.Length} of resource id {FhirBundleParser.ExtractResourceId(jsonObject)} is larger than {LargeResourceSize} Bytes.");
```

Comment:
Maybe also log the Resource Type?

Reply:
I just found we only log "Schema Type" in current **CheckBlockSize**, the resource Id is not contained, maybe follow that behavior?

Reply:
so, need we log resource type or schema type here?

Reply:
per our discussion, removed the resource id and use schema type and resource type in the log